streams/src/test/java/org/apache/kafka/streams/integration/utils/IntegrationTestUtils.java

```diff
@@ -520,6 +662,33 @@ public static void waitUntilMetadataIsPropagated(final List<KafkaServer> servers
         return consumedValues;
     }
 
+    public static KafkaStreams getCleanStartedStreams(final String broker, final String appId, final boolean eosEnabled, final StreamsBuilder builder) {
```

Comment:
seems like the underlying behavior `getCleanStartedStreams` and  `getStartedStreams` methods could be consolidated into a single method taking a `boolean` for a clean start or not. The original methods would simply delegate to the new method and pass the boolean flag.

Reply:
I was on the fence about whether to make two methods or have a flag at the top level anyway. It sounds like you'd prefer a flag, so I've gone ahead and done it that way instead.